sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManagerFactoryProvider.java

```diff
@@ -30,7 +30,7 @@
     /**
      * Stores the version.
      */
-    private static final Double VERSION = 1.0;
```

Comment:
Made this change so we wouldn't use a deprecated API in line 39.

Reply:
Same here.